spec/models/google_calendar/event_wrapper/rounded_event_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+describe GoogleCalendar::EventWrapper::RoundedEvent do
```

Comment:
This one deserves a good refactor :p 

You can start with extracting `event`, `start`, `end`, `params` lets to describe block and put only things that differ into subcontexts.